log_img_pub:
  ros__parameters:
    folder_path: "/home/chiba/share/camera_lidar_data/2023_11_04_15_22_48/2023_11_04_15_22_48/camera1"
